device/arista/x86_64-arista_7060x6_16pe_384c_b/Arista-7060X6-16PE-384C-B-O128S2-FANOUT/sai.profile

```
@@ -1,3 +1,4 @@
 SAI_INIT_CONFIG_FILE=/usr/share/sonic/hwsku/th5-a7060x6-16pe-384c.config.bcm
 SAI_NUM_ECMP_MEMBERS=128
 SAI_NHG_HIERARCHICAL_NEXTHOP=false
+SAI_SWITCH_SWITCHING_MODE_CUT_THROUGH=true
```

Why `CUT_THROUGH` is enabled? Is that expected?

---

Will this cause packet with CRC being propagated to neigh devices?

---

A follow up question, do we have test to ensure CRC error is not propagated?